### PR TITLE
Add SDL log verbosity conf.

### PR DESCRIPTION
**Pull request by toots**
This PR adds SDL log verbosity conf keys. However, they are not called early enough and I still see annoying logs when loading SDL stuff.

### CODE REVIEW

**Comment by smimram** [comment]
Note for later, the first print is there:
```
#0  __GI___libc_write (fd=2, buf=0x7fffffffd4b0, nbytes=40) at ../sysdeps/unix/sysv/linux/write.c:26
#1  0x00007ffff3c9dbb5 in _IO_new_file_write (f=0x7ffff3df26a0 <_IO_2_1_stderr_>, data=0x7fffffffd4b0, n=40) at ./libio/fileops.c:1180
#2  0x00007ffff3c9cf70 in new_do_write (fp=fp@entry=0x7ffff3df26a0 <_IO_2_1_stderr_>, data=data@entry=0x7fffffffd4b0 "INFO: Loading Sdl_image, Target = linux\np", to_do=to_do@entry=40) at ./libio/libioP.h:946
#3  0x00007ffff3c9e2a1 in _IO_new_file_xsputn (n=40, data=<optimized out>, f=0x7ffff3df26a0 <_IO_2_1_stderr_>) at ./libio/fileops.c:1254
#4  _IO_new_file_xsputn (f=0x7ffff3df26a0 <_IO_2_1_stderr_>, data=<optimized out>, n=40) at ./libio/fileops.c:1196
#5  0x00007ffff3c71409 in __printf_buffer_flush_to_file (buf=buf@entry=0x7fffffffd480) at ../libio/libioP.h:946
#6  0x00007ffff3c714c0 in __printf_buffer_to_file_done (buf=buf@entry=0x7fffffffd480) at ./stdio-common/printf_buffer_to_file.c:120
#7  0x00007ffff3c7ac0d in __vfprintf_internal (s=0x7ffff3df26a0 <_IO_2_1_stderr_>, format=0x7ffff7398653 "%s: %s\n", ap=ap@entry=0x7fffffffd580, mode_flags=2) at ./stdio-common/vfprintf-internal.c:1475
#8  0x00007ffff3d34a1f in ___fprintf_chk (fp=<optimized out>, flag=<optimized out>, format=<optimized out>) at ./debug/fprintf_chk.c:33
#9  0x00007ffff72583bb in  () at /lib/x86_64-linux-gnu/libSDL2-2.0.so.0
#10 0x00007ffff7276af4 in SDL_Log () at /lib/x86_64-linux-gnu/libSDL2-2.0.so.0
#11 0x00007ffff721df7a in  () at /lib/x86_64-linux-gnu/libffi.so.8
#12 0x00007ffff721d40e in  () at /lib/x86_64-linux-gnu/libffi.so.8
#13 0x00007ffff721db0d in ffi_call () at /lib/x86_64-linux-gnu/libffi.so.8
#14 0x0000555556061681 in ctypes_call (fnname=<optimized out>, function=<optimized out>, callspec_=<optimized out>, argwriter=<optimized out>, rvreader=<optimized out>)
    at /home/smimram/.opam/5.0.0/.opam-switch/build/ctypes.0.20.2/src/ctypes-foreign/ffi_call_stubs.c:404
#15 0x00005555560d971b in <signal handler called> ()
#16 0x0000555555b2d180 in camlCtypes_ffi__fun_1348 () at src/ctypes-foreign/ctypes_ffi.ml:111
#17 0x0000555555acef6f in camlTsdl__fun_10326 () at src/tsdl.ml:299
#18 0x0000555555ac4886 in camlTsdl_image__entry () at src/tsdl_image.ml:45
#19 0x0000555555a4e71b in caml_program ()
#20 0x00005555560d97d4 in <signal handler called> ()
#21 0x00005555560d918f in caml_startup_common (pooling=<optimized out>, argv=0x7fffffffdda8) at runtime/startup_nat.c:129
#22 caml_startup_common (argv=0x7fffffffdda8, pooling=<optimized out>) at runtime/startup_nat.c:85
#23 0x00005555560d920b in caml_startup_exn (argv=<optimized out>) at runtime/startup_nat.c:136
#24 caml_startup (argv=<optimized out>) at runtime/startup_nat.c:141
#25 caml_main (argv=<optimized out>) at runtime/startup_nat.c:148
#26 0x0000555555a4be5c in main (argc=<optimized out>, argv=<optimized out>) at runtime/main.c:37
```

**Comment by smimram** [comment]
Reported here: https://github.com/sanette/tsdl-image/issues/8

File: src/core/tools/sdl_utils.ml [hunk 1]
```diff
@@ -39,11 +83,20 @@ let options : Sdl.Init.t option ref = ref None
 let check f x =
   match f x with Error (`Msg err) -> failwith err | Ok ans -> ans
 
-let initialized = ref false
+(* This is also done is sdl_log_level, but just in case... *)
```
Comment (toots) [inline src/core/tools/sdl_utils.ml:86]:
```suggestion
(* This is also done in sdl_log_level, but just in case... *)
```

**Review by smimram — APPROVED** [review]

**Review by toots — COMMENTED** [review]
giphy (1)

**Pull request merged**